itests/hive-unit/src/test/java/org/apache/hive/jdbc/BaseJdbcWithMiniLlap.java

```diff
@@ -530,12 +498,10 @@ protected int processQuery(String currentDatabase, String query, int numSplits,
     InputSplit[] splits = inputFormat.getSplits(job, numSplits);
 
     // Fetch rows from splits
-    boolean first = true;
```

Comment:
These changes do not seem related to this patch? If they are not, can we submit it in a different one?